test/test_env_files.py

```diff
@@ -238,26 +238,26 @@ def test_permission_denied_handling(
     tmp_path: Path, clean_env: pytest.MonkeyPatch
 ) -> None:
     """Test that permission denied errors are handled gracefully."""
-    pytest.importorskip("dotenv")
+    dotenv = pytest.importorskip("dotenv")
+    from unittest.mock import patch
```

Comment:
Did you want to include this on the top of the imports? (since built-in)

Reply:
ugh why does AI do that thanks